src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1066,45 +1034,43 @@ public ConferenceShim getShim()
      * Broadcasts the packet to all endpoints and tentacles that want it.
      *
      * @param packetInfo the packet
-     * @param source the source endpoint, or {@code null} to indicate that the
-     * packet comes from another bridge via Octo.
+     * @param source the ID of the source endpoint.
      */
-    private void sendOut(PacketInfo packetInfo, AbstractEndpoint source)
+    private void sendOut(PacketInfo packetInfo, String source)
```

Comment:
if `source` is going to be a string, can we rename it to something less ambiguous? `sourceEpId`?

Reply:
I specifically renamed all of them to `source` because it's clear from the type what it is, and I don't think the longer names add anything here.